js/exercise-04.js

```diff
@@ -0,0 +1,18 @@
+/*
+    What would each of these lines of code do:
+
+    1. true && console.log("moo");
+    2. false && console.log("moo moo?")
+    3. true || console.log("hello friend");
+    4. false || console.log("bye friend");
+
+    Once you see the results, write in a comment why it works that way
+*/
+
+
+//RESPONSE:
+
+true && console.log("moo");  
```

Comment:
This exercise is not complete.
```suggestion
true && console.log("moo");  // output + explanation
```

FIX: Execute the script and add a comment to each statement explaining its output and why it happens.